docs/team/joel.md

```diff
@@ -2,5 +2,17 @@
 
 ## Overview
 
+//TODO Standardise
 
 ### Summary of Contributions
+
+- New Feature: Implemented the ability to mark existing tasks in the tasklist as done.
+- New Feature: Implemented a personalised message generator that encourages the user after certain successful input.
+- Code Contributed: [Reposense link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=yAOwzers&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+- Documentation: 
+    - User Guide: 
+        - Included details for `mark as done` feature. [#84](https://github.com/AY2021S1-CS2113T-T12-3/tp/pull/84)
+    - Developer Guide:
+        - Added implementation details of the `mark as done` feature. [#145](https://github.com/AY2021S1-CS2113T-T12-3/tp/pull/145)
+        - Added manual test cases under `appendix F`. [#129](https://github.com/AY2021S1-CS2113T-T12-3/tp/pull/129)
+    
```

Comment:
Include some of the PRs you reviewed and bugs you reported of a grp

Reply:
Oh why thank you will do